src/raw/http_util/uri.rs

```diff
@@ -12,10 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use crate::{Error, ErrorKind, Scheme};
 use percent_encoding::utf8_percent_encode;
 use percent_encoding::AsciiSet;
 use percent_encoding::NON_ALPHANUMERIC;
 
+/// Result that is a wrapper of `Result<T, opendal::Error>`
+pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
**suggestion**: Maybe we can use `crate::Result` directly?

Reply:
Yes, we can. I just see other file do in this way, so I follow original style